src/js/modules/RacingData.js

```diff
@@ -0,0 +1,16 @@
+const racingData = {
+  status: [],
+  goalTryNumber: 0,
+  initialize() {
+    this.status = [];
+    this.goalTryNumber = 0;
+  },
+  set raceReadyStatus(carNamesArray) {
+    this.status = carNamesArray.map((name) => ({ name, position: 0 }));
+  },
```

Comment:
```suggestion
  setRaceReadyStatus(carNamesArray) {
    this.status = carNamesArray.map((name) => ({ name, position: 0 }));
  },
```
이건 setter 대신에 method로 표현하는게 명시적이지 않을까요?